### DISCUSSION TITLE
추가적인 오픈소스 정보 추가

### DISCUSSION THREAD

**Discussion by hololee** [mlops-for-all/mlops-for-all.github.io#74] [Ideas]
<img width="1076" alt="image" src="https://user-images.githubusercontent.com/30971523/148196367-f36a03a7-a587-4eb2-8c97-52320bf51839.png">

여기서 보시면 추가적으로 보면 좋을 오픈소스를 정리해서 그림으로 올려주셨는데
오픈소스의 아이콘을 보고 어떤 키워드로 검색을 해야할지 알지 못해서 텍스트로 오픈소스 이름을 적어주신다면 엄청 큰 도움이 될것 같습니다!

어디에다가 전달할까 하다가 이슈로 남깁니다 ㅠ

p.s 이슈로 남겼다가 여기로 옮기는게 좋을거 같아서 다시 남깁니다..ㅠ

**Comment by anencore94** [comment]
안녕하세요! 기여 감사드립니다. 미처 고려하지 못했던 정말 좋은 의견인 것 같습니다! 곧 반영하도록 하겠습니다!

**Comment by Aiden-Jeon** [comment]
안녕하세요!
추가적인 오픈소스들의 이름을 정리하는 표를 추가한 [PR](https://github.com/mlops-for-all/mlops-for-all.github.io/pull/75)을 날렸습니다.
감사합니다.

**Reply by hololee** [reply]
감사합니다!!!!!

**Reply by Aiden-Jeon** [reply]
이제 홈페이지에서 확인하실 수 있습니다!
https://mlops-for-all.github.io/docs/further-readings/info/
